### PR TITLE
Create config file for phpunit and improve docs

### PR DESCRIPTION
**Pull request by cpeel**
Creates a configuration file for phpunit that specifies the bootstrap settings we need. Small improvements to various README files and comments around testing as well.

### CODE REVIEW

File: SETUP/tests/README.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-## Running tests
+The phpunit tests, as-coded, will work with phpunit 6.x, 7.x, 8.x, and 9.x.
 
-To run them, use the `phpunit_bootstrap.php` file to pull in pre-requisits.
+### Running tests
 
-    phpunit --bootstrap phpunit_bootstrap.php --verbose .
+The `phpunit.xml` contains configuration information like bootstrap
+requirements, so running the full suite is as simple as:
+```bash
```
Comment (chrismiceli) [inline SETUP/tests/README.md:20]: I assume it works on other shells too :-)

File: SETUP/tests/phpunit.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap = "phpunit_bootstrap.php"
+    colors        = "false"
+    stopOnFailure = "false"
+    verbose       = "false">
```
Comment (chrismiceli) [inline SETUP/tests/phpunit.xml:6]: We had verbose on before, no?

**Review by chrismiceli — APPROVED** [review]

File: SETUP/tests/README.md (hunk 1 above)
Comment (cpeel) [reply SETUP/tests/README.md:20]: Ha! Yes - this is just MarkDown sugar to colorize it as a shell script.

**Review by chrismiceli — APPROVED** [review]

File: SETUP/tests/phpunit.xml (hunk 2 above)
Comment (cpeel) [reply SETUP/tests/phpunit.xml]: Oh, good catch, yes we did. Fixed.

**Review by srjfoo — APPROVED** [review]

**Review by 70ray — APPROVED** [review]

**Pull request merged**